ci/cloudbuild/triggers/quickstart-bazel-ci.yaml

```diff
@@ -7,4 +7,6 @@ github:
 name: quickstart-bazel-ci
 substitutions:
   _BUILD_NAME: quickstart-bazel
-  _DISTRO: ubuntu-bionic
+  _DISTRO: fedora
```

Comment:
nit: this seems unrelated, I am Okay merging it now.  If it was not intentional, please fix. If it was intentional, consider separating to another PR (but again, I am Okay either way).

Reply:
These `_DISTRO` changes were all accidental. Good catch! All fixed/reverted back to `ubuntu-bionic` in the places that changed.